### PR TITLE
add info about testing with airborne

### PR DESCRIPTION
**Pull request by sethpollack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2306,6 +2306,46 @@ describe Twitter::API do
   end
 end
 ```
+####Writing Tests with Airbrone
```
Comment (dblock) [inline README.md]: Missing space after ####, spelling mistake in Airborne, and it should be consistent with above, so just Airborne, not "Wirting Tests with Airborne".

File: README.md [hunk 2]
```diff
@@ -2306,6 +2306,46 @@ describe Twitter::API do
...
+
+Alternativly you can test with [Airbone](https://github.com/brooklynDev/airborne). Under the covers, `Airborne`   uses `rack-test` to make the requests. It also parses the response and adds a bunch of helpers for testing json, including path matching for testing nested json and arrays.
```
Comment (dblock) [inline README.md]: Remove "Alternatively"

File: README.md [hunk 3]
```diff
@@ -2306,6 +2306,46 @@ describe Twitter::API do
...
+  config.rack_app = MyGrapeApp
+end
+```
+
+```ruby
+require 'spec_helper'
+
+describe 'sample spec' do
+
+  it 'should validate types' do
```
Comment (dblock) [inline README.md]: 'should validate types' => 'validates types'

**Comment by dblock** [comment]
There're a lot of spelling mistakes and too much detail, for this to be merged, please reduce this to a bare minimum of how to set this up with Airborne.

Generally though there's really nothing special about Airborne, it's a Rack::Test-based thing, so I am not convinced we need this.

**Comment by dm1try** [comment]
>Generally though there's really nothing special about Airborne, it's a Rack::Test-based thing

@dblock it's because `airborne` uses `Rack::Test` :) I'm agree, just mention `airborne` is enough

**Comment by sethpollack** [comment]
Ok, I cleaned it up. Sorry about that!

**Comment by dblock** [comment]
I still wonder whether we really need this. Airborne is RSpec-driven, it's basically RSpec with some helpers (unless I am missing something) and itself explains very clearly how to work with Rack-based applications. 

I would like to hear from others whether we want this in the README.

File: README.md [hunk 4]
```diff
@@ -2306,6 +2306,34 @@ describe Twitter::API do
   end
 end
 ```
+#### Airborne
+
+You can test with [Airborne](https://github.com/brooklynDev/airborne) which uses `rack-test` to make requests. It then parses the response and provides helpers for testing JSON, including path matchers for testing nested JSON and arrays.
+
+```ruby
+#spec_helper.rb
```
Comment (dblock) [inline README.md]: Commented spec-helper? Remove.

File: README.md [hunk 5]
```diff
@@ -2306,6 +2306,34 @@ describe Twitter::API do
...
+require 'airborne'
+Airborne.configure do |config|
```
Comment (dblock) [inline README.md]: Space above this.

File: README.md [hunk 6]
```diff
@@ -2306,6 +2306,34 @@ describe Twitter::API do
...
+  config.rack_app = MyGrapeApp
+end
+```
+
+```ruby
+require 'airborne'
+
+describe 'sample spec' do
+  it 'validates types' do
+    get '/api/v1/simple_get' #json api that returns { "name" : "John Doe" }
```
Comment (dblock) [inline README.md]: Make this as if Rubocop ran on it, so space after pound, `# json ...`, then there're spaces between ` { }` and the hash syntax below should be Ruby 1.9.3, so `name: 'John Doe'`.

Comment (dblock) [inline README.md]: Please also squash the commits.

**Comment by sethpollack** [comment]
Ok, I ran RuboCop and squashed the commits.

**Comment by dm1try** [comment]
@dblock, agree! A single line in RSpec section is enough I think.
"You can easily test your JSON API with `airbone` - RSpec-driven framework which provide a bunch of helpful helpers" :) or smth similar

**Comment by dblock** [comment]
I rewrote this to use the Twitter::API class as every other example in the README, shortened, and merged via https://github.com/intridea/grape/commit/30aa864b625fe9208d37ce50bc759cfd1ccc34aa.

#### Airborne

You can test with other RSpec-based frameworks, including Airborne, which uses rack-test to make requests.

```ruby
require 'airborne'

Airborne.configure do |config|
  config.rack_app = Twitter::API
end

describe Twitter::API do
  describe "GET /api/statuses/:id" do
    it "returns a status by id" do
      status = Status.create!
      get "/api/statuses/#{status.id}"
      expect_json(status.as_json)
    end
  end
end
```

**Comment by sethpollack** [comment]
Thanks!

**Pull request closed without merging**